S/SPIRV_LLVM_Backend/build_tarballs.jl

```diff
@@ -0,0 +1,93 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SPIRV_LLVM_Backend"
+version = v"19"
+
+# Collection of sources required to build SPIRV_LLVM_Backend
+sources = [
+    # LLVM v19.1.7
+    GitSource("https://github.com/llvm/llvm-project",
```

Comment:
```suggestion
    GitSource("https://github.com/llvm/llvm-project.git",
```
we should have this repo already cached from https://github.com/JuliaPackaging/Yggdrasil/blob/52fe3463e71686c37e2dbc97ffc0d1019fe99d36/L/LLVMLibcxx/build_tarballs.jl#L9 (although I think using the tarball would be more efficient, cloning llvm with libgit2 takes forever)

Reply:
Good call, I'll update it to the tarball.